deploy/src/main/deploy/example-credentials.json

```diff
@@ -40,6 +40,26 @@
             "pwd-hash": "CrHirmXdU20qORDRE5gbphS9vUCdl2NhEaAikw6EGTwBevbs1rCwml82cONPNxugZ1D0QHQSnbY4gWJQJi6P4g=="
           }
         ]
+      },
+      {
+        "device-id" : "4711",
+        "type" : "psk",
+        "auth-id" : "sensor1",
+        "enabled" : true,
+        "secrets" : [
+          {
+            "not-before": "2018-01-01T00:00:00+01:00",
+            "not-after": "2037-06-01T14:00:00+01:00",
+            "comment": "coap preshared key: coap-secret",
+            "key" : "Y29hcC1zZWNyZXQ="
+          }
+        ]
+      },
+      {
+        "device-id": "4711",
+        "type": "x509-cert",
+        "auth-id": "C=CA,L=Ottawa,O=Eclipse IoT,OU=Californium,CN=cf-client",
```

Comment:
this doesn't seem to be related to PSK auth, is it?

Reply:
Sure, I remove it.